raiden/raiden_service.py

```diff
@@ -212,14 +220,41 @@ def update_monitoring_service_from_balance_proof(
     if raiden.config['services']['monitoring_enabled'] is False:
         return
 
+    channel_state = views.get_channelstate_by_canonical_identifier(
+        chain_state=views.state_from_raiden(raiden),
+        canonical_identifier=CanonicalIdentifier(
+            chain_identifier=raiden.chain.network_id,
+            token_network_address=new_balance_proof.token_network_identifier,
+            channel_identifier=new_balance_proof.channel_identifier,
+        ),
+    )
+
+    msg = (
+        f'Failed update monitoring service due to failing to find '
```

Comment:
super nitpick on the wording: `due to inability to find` just so you don't repeat "fail"